<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppToolClass</class>
 <widget class="QDialog" name="AppToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AppTool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="sourceLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="sourceFileButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>源目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="targetFileButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标目录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modelComboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="updateGroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>升级模式</string>
   </property>
   <widget class="QRadioButton" name="allRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>全部</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="settingRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="compareRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>125</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>比较</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>updateGroupBox</zorder>
  <zorder>sourceLineEdit</zorder>
  <zorder>sourceFileButton</zorder>
  <zorder>targetLineEdit</zorder>
  <zorder>targetFileButton</zorder>
  <zorder>modelComboBox</zorder>
  <zorder>startButton</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="apptool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sourceFileButton</sender>
   <signal>clicked()</signal>
   <receiver>AppToolClass</receiver>
   <slot>selectSourceDictionary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>targetFileButton</sender>
   <signal>clicked()</signal>
   <receiver>AppToolClass</receiver>
   <slot>selectTargetDictionary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>AppToolClass</receiver>
   <slot>startApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>AppToolClass</receiver>
   <slot>exitApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectSourceDictionary()</slot>
  <slot>selectTargetDictionary()</slot>
  <slot>startApp()</slot>
  <slot>exitApp()</slot>
 </slots>
</ui>
